src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/records/CalledLegacySegment.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.records;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+public class CalledLegacySegment extends LegacySegment {
+
+    private CalledCopyRatioSegment.Call call;
+
+    public CalledLegacySegment(final String sampleName, final SimpleInterval interval, final int numProbes, final double segmentMean,
+                               final CalledCopyRatioSegment.Call call) {
+        super(sampleName, interval, numProbes, segmentMean);
+        this.call = call;
+    }
+
+    public CalledCopyRatioSegment.Call getCall() {
+        return call;
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) return true;
```

Comment:
Clean up these auto-generated methods by adding braces and `final` and removing white space.

Reply:
Also, make sure to update these methods if `call` is required to be non-null.

Reply:
Done